### PR TITLE
feat: toPercent()

### PR DESCRIPTION
**Pull request by ianzone**
this function converts a fraction number to a string in percentage format.

### CODE REVIEW

**Comment by UlisesGascon** [comment]
I will suggest to include tests, [here](https://github.com/lodash/lodash/blob/main/test/inRange.spec.js) you can find an example from the `inRange()` function ([docs](https://lodash.com/docs/4.17.15#inRange)).

**Comment by ianzone** [comment]
I just added the test.

**Comment by ianzone** [comment]
Hi, is it ok?

File: test/toPercent.spec.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import toPercent from '../src/toPercent';
+
+describe('toPercent', () => {
```
Comment (UlisesGascon) [inline test/toPercent.spec.ts:3]: I will suggest to include tests to cover the usage of `minPrecision` and `maxPrecision`.

File: src/toPercent.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * // => 20%
+ *
+ * toPercent(0.3333333)
+ * // => 33.33%
+ *
+ * toPercent(0.2, 2)
+ * // => 20.00%
+ *
+ */
+export default function toPercent(fraction: number, minPrecision: number = 0, maxPrecision: number = 2) {
```
Comment (UlisesGascon) [inline src/toPercent.ts:21]: Note  that `minPrecision` and `maxPrecision` are not used, so this example generates an unxpected output:

```
toPercent(12, 4, 7)
// '1,200%'
```

IMO, seems like the output is very deterministic as the locals are not included, ref https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat#using_locales.

**Pull request closed without merging**